### PR TITLE
Clicking of some areas of the panel does not gain focus to the panel

### LINKED ISSUE #512: Clicking of some areas of the panel does not gain focus to the panel
For example, clicking on the area indicated below does not make the panel gain focus
image

### PR DESCRIPTION
**Pull request by rd1992**
Fixes #512
Also added css effect to highlight selected column

### CODE REVIEW

File: src/ui/issuecolumn/IssueColumn.java [hunk 1]
```diff
@@ -68,6 +70,21 @@ public IssueColumn(UI ui, Stage mainStage, Model model, ColumnControl parentColu
...
+			public void changed(ObservableValue<? extends Boolean> unused, Boolean wasFocused, Boolean isFocused) {
+				if (!isFocused) {
+					if (getStyleClass().contains("issue-panel-focused"))
+						getStyleClass().remove("issue-panel-focused");
+					return;
+				}
+				getStyleClass().add("issue-panel-focused");
+			}
+		});
 	}
```
Comment (dariusf) [inline src/ui/issuecolumn/IssueColumn.java]: Is there a need to check for the presence of the style class, i.e. could the following work?

```java
if (isFocused) {
    getStyleClass().add("issue-panel-focused");
} else {
    getStyleClass().remove("issue-panel-focused");
}
```

Comment (rd1992) [inline src/ui/issuecolumn/IssueColumn.java]: not really. just being defensive. if by some odd reason that css rule is set somewhere else.

Comment (dariusf) [inline src/ui/issuecolumn/IssueColumn.java]: Maybe use an assertion? So if the style class gets on there for some reason we would be alerted.

Comment (rd1992) [inline src/ui/issuecolumn/IssueColumn.java]: actually i realized no need for an assertion also. the remove() function does not throw exceptions if the element is not present. so no need for the if check

**Pull request merged**